### DISCUSSION TITLE
Relationship value reset

### DISCUSSION THREAD

**Discussion by juangbm** [Budibase/budibase#9362] [Help] [answered]
Hi,
I have a form for updating information where I have some relationship fields, by default I have edition disabled, but by clicking on Edit all fields are updatable, in addition when the Edit button is clicked, I enable a Cancel button, just in case the user wants to restore initial information. I can restore all fields except Relationship Fileds ... Any idea how can I re-display the initial information for the relationship fields?

**Comment by andz-bb** [comment, marked as answer]
Hey @juangbm, how do you have your readonly/edit modes set up? I made an example app using a standard table block and added an edit/view button to change whether the form is disabled or not. When I toggle between the states the relationship data is preserved.

![toggle view/edit relationship data](https://user-images.githubusercontent.com/110921612/213261840-c366e229-2274-4f3d-a732-2873a00cb507.gif)

Here's an app export if you want to have a closer look at how I'm doing this:
[display or edit-export-1674066223667.tar.gz](https://github.com/Budibase/budibase/files/10449446/display.or.edit-export-1674066223667.tar.gz)

If you would like me to have a look at your specific implementation feel free to send an app export to support@budibase.com or [on our Discord server here](https://discord.gg/ZepTmGbtfF). Thanks!

**Reply by juangbm** [reply]
Thanks! This solves the issue!!
